cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -83,9 +85,10 @@
     private Boolean taintCheck;
 
     private static final String runCmd = "bal run [--experimental] [--offline] [--taint-check]\n" +
-            "                  <executable-jar | ballerina-file | . | package-path> [program-args] [(--key=value)...]";
+            "                  [<executable-jar | ballerina-file | package-path>] [-- program-args...]";
```

Comment:
How would the -- works for executable jar file since we short circuit its execution via bash script

Reply:
It won't work since it's executed via the bash script. We used to have a similar inconsistency of not being able to pass configs when executable jar is provided. This would be addressed by the last point of ballerina-platform/ballerina-spec#764.

`Remove bal run <path-to-jar> and add a new command bal java <path-to-jar>`

Can we go with this limitation till we make the mentioned improvement?